app/models/occupation_standard.rb

```diff
@@ -16,9 +16,24 @@ class OccupationStandard < ApplicationRecord
 
   delegate :title, to: :organization, prefix: true
   delegate :title, to: :occupation, prefix: true
-  delegate :title, to: :industry, prefix: true
+  delegate :title, to: :industry, prefix: true, allow_nil: true
   delegate :name, to: :creator, prefix: true
 
+  class << self
+    def filter_collection(options={})
+      options.delete_if { |k, v| v != false && v.blank? }
+      options.delete_if { |k, v| v.kind_of?(Array) and v.reject(&:blank?).empty? }
+      options.inject(all) do |scope, (key, value)|
```

Comment:
Just a style comment on this. We do something similar in workhands with these checks for blanks/empties but push it out to scopes and simply chain the scopes together. 

No need to change anything here, but keep it in mind as another pattern if this needs to support a bunch more options in the `case` statement.

https://github.com/WorkHands/WorkHands/blob/master/engines/apprenticeship/app/models/apprenticeship/search/review.rb

https://github.com/WorkHands/WorkHands/blob/master/engines/apprenticeship/app/models/apprenticeship/review.rb#L36